<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QToolButton" name="optionsButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Plant Watering and Monitoring System</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3" colspan="2">
     <layout class="QGridLayout" name="optionsMenu" rowstretch="0">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="optionsMenuFrame">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="optionsLabel">
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Light</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Moisture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Temperature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Humidity</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>771</width>
           <height>451</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="networkingLabel">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Networking</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="disconnectPlantButton">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>Disconnect Plant</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="forgetPlantButton">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>Forget Plant</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="preferencesLabel">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Preferences</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="darkModeCheckBox">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>Dark Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="hideOnCloseCheckBox">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>Hide on Close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="sendNotificationsCheckBox">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>Send Notifications</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
